lib/mix/tasks/vbt/bootstrap.ex

```diff
@@ -11,5 +11,47 @@ defmodule Mix.Tasks.Vbt.Bootstrap do
       ~w/makefile docker circleci heroku github_pr_template credo dialyzer/,
       &Mix.Task.run("vbt.gen.#{&1}", args)
     )
+
+    Mix.shell().info(manual_instructions())
+  end
+
+  defp manual_instructions do
+    """
+    You also need to make the following changes in your mix.exs file:
+
+        def project do
+          [
+            # ...
+            preferred_cli_env: preferred_cli_env(),
+            dialyzer: dialyzer()
+          ]
+        end
+
+        def deps do
+          [
+            # ...
+            {:dialyxir, "~> 0.5", runtime: false}
```

Comment:
Do we add `only: [:dev, :test]` here? Saw it on DMF and here, seems prod never needs this.

Reply:
We should be able to run dialyzer on a prod-compiled version, so dialyxir should exist in all envs. The `runtime: false` flag ensures that dialyxir will not be included in the OTP release.